templates/default/appserver.nginx.conf.erb

```diff
@@ -140,6 +152,18 @@ server {
     add_header Access-Control-Expose-Headers ETag;
   }
 
+  <% if @out[:acao_packs] %>
+    location ^~ /packs/ {
+    gzip_static on;
+    expires max;
+    add_header Cache-Control public;
+    add_header Access-Control-Allow-Origin "$http_origin";
+    add_header Access-Control-Allow-Methods 'GET, PUT, POST, DELETE';
+    add_header Access-Control-Expose-Headers ETag;
```

Comment:
Noticing that the SSL version of the `location ^~ /assets/` directive didn't include `X-Content-Type-Options nosniff`. Any reason that it was omitted here?

Reply:
Yes - oversight :) Thanks, for catching this up, I'll submit a fix.